src/Backend/Session.vala

```diff
@@ -252,6 +254,12 @@ public class Mail.Backend.Session : Camel.Session {
         return service;
     }
 
+    public override void remove_service (Camel.Service service) {
+        var account = new Account (service);
```

Comment:
I'm not sure we need to construct an `Account` here and pass it as a parameter to the signal as it's never used.

Reply:
The reason I allready put it in here is to make sure the signal can be used in other places. For example the account list could use this to remove an account when necessary, but to do that it needs to know which account to remove. Should i be proactive and put it in there, or leave it out until it is necessary for another usecase?

Reply:
I'd say leave it out until it is necessary. Since you're constructing a new object here, it's not the same `Account` as one constructed elsewhere, so it may not be that useful for comparisons to already existing `Account` objects.

Reply:
OK, I have removed the parameter from the signal. Thanks for the thorough review